docs/snapshotting/snapshot-support.md

```diff
@@ -638,28 +640,19 @@ might not be able to handle the injected notification and crash. We suggest to
 users that they take snapshots only after the guest kernel has completed
 booting, to avoid this issue.
 
-## Snapshot compatibility across kernel versions
+## Where can I resume my snapshots?
 
-We have a mechanism in place to experiment with snapshot compatibility across
-supported host kernel versions by generating snapshot artifacts through
-[this test](../../tests/integration_tests/functional/test_snapshot_phase1.py)
-and checking devices' functionality using
-[this test](../../tests/integration_tests/functional/test_snapshot_restore_cross_kernel.py).
-The test restores the snapshot and ensures that all the devices set-up (network
-devices, disk, vsock, balloon and MMDS) are operational post-load.
+Snapshots must be resumed on an software and hardware configuration which is
+identical to what they were generated on. However, in limited cases, snapshots
+can be resumed on identical hardware instances where they were taken on, but
+using newer host kernel versions. While we do not provide any guarantees on this
+setup (and do not recommend doing this in production), we are currently aware of
```

Comment:
nit: can remove parentheses

Reply:
I like parenthesis though :(

since you approved I'll leave it xP